### PR TITLE
[DNM] Debug netlify/hugo timeout issue

### PR DESCRIPTION
**Pull request by tengqm**

### CODE REVIEW

File: content/en/_index.html [hunk 1]
```diff
@@ -7,53 +7,63 @@
...
+Whether testing locally or running a global enterprise, Kubernetes flexibility grows with you to deliver your
+applications consistently and easily no matter how complex your need is.
 
 {{% /blocks/feature %}}
 
 {{% blocks/feature image="suitcase" %}}
 #### Run K8s Anywhere
 
-Kubernetes is open source giving you the freedom to take advantage of on-premises, hybrid, or public cloud infrastructure, letting you effortlessly move workloads to where it matters to you.
+Kubernetes is open source giving you the freedom to take advantage of on-premises, hybrid, or public cloud
```
Comment (chrisnegus) [inline content/en/_index.html:41]:
```suggestion
Kubernetes is open source, giving you the freedom to take advantage of on-premises, hybrid, or public cloud
```

**Review by chrisnegus — COMMENTED** [review]
One small suggestion.

File: content/en/_index.html [hunk 2]
```diff
@@ -8,52 +8,61 @@
...
+Whether testing locally or running a global enterprise, Kubernetes flexibility grows with you to deliver your
+applications consistently and easily no matter how complex your need is.
 
 {{% /blocks/feature %}}
 
 {{% blocks/feature image="suitcase" %}}
 #### Run K8s Anywhere
 
-Kubernetes is open source giving you the freedom to take advantage of on-premises, hybrid, or public cloud infrastructure, letting you effortlessly move workloads to where it matters to you.
+Kubernetes is open source,giving you the freedom to take advantage of on-premises, hybrid, or public cloud
```
Comment (divya-mohan0209) [inline content/en/_index.html:40]: Non-blocking nit.
```suggestion
Kubernetes is open source, giving you the freedom to take advantage of on-premises, hybrid, or public cloud
```

**Review by divya-mohan0209 — DISMISSED** [review]
A small nit.

**Comment by tengqm** [comment]
The timeout happens at the same file again:

```
10:52:39 PM: DEBUG 2023/01/08 14:52:17 Render page Відкриття доступу до застосунку за межами кластера to "/uk/docs/tutorials/kubernetes-basics/expose/index.html"
11:25:45 PM: Build exceeded maximum allowed runtime
```

The build hangs when processing an `index.html` file in `uk` localization. Hopefully, this can be resolved after #38834 is in.

File: content/en/_index.html (hunk 2 above)
Comment (tengqm) [reply content/en/_index.html:40]: This PR is not meant to be merged. It is not meant to change these files at all.
A trivial change is incorporated to make sure that there is something new to trigger a build.

**Comment by tengqm** [comment]
Triggered another timeout. This time, the file processing times out at `content/pl/training/_index.html` file.

```
5:22:35 PM: DEBUG 2023/01/30 09:22:14 Render page Szkolenia to "/pl/training/index.html"
5:55:45 PM: Build exceeded maximum allowed runtime
```

**Comment by tengqm** [comment]
Build hang on another file:

```
7:35:29 PM: DEBUG 2023/02/02 11:35:05 Render page kubectl CLI to "/es/docs/reference/kubectl/index.html"
```

**Comment by kbhawkey** [comment]
👋 @tengqm .
Do you want to continue with these change to debug the build?

**Comment by tengqm** [comment]
> 👋 @tengqm . Do you want to continue with these change to debug the build?

Em ... It seems not related to Hugo, so this can be left as a WIP. I am still struggling with the root cause of build hang. Turning on Netlify debug mode didn't give us anything useful either.

**Comment by sftim** [comment]
Is this now OK to close?

**Pull request closed without merging**